lib/tasks/reindex.rake

```diff
@@ -1,6 +1,13 @@
-desc 'Reindex all exhibit items'
-task reindex: :environment do
+desc 'Reindex all exhibit items synchronously'
+task reindex_now: :environment do
```

Comment:
One of these probably should be `reindex`, yes?

Reply:
Good point. I've made `perform_later` the new default for that.